### PR TITLE
fix: Custom MailObject SendFrom Vs Replyto

### PR DESCRIPTION
**Pull request by DerekCrannaford**
Check & patch anywhere a custom mailer bypasses transport->send
hence can be setting it's own 'from'
Make sure 'from' is 'company'
(Also patch for cron action fall through to default .tpl on ShowTimer)

refs: Client post SES transition
issues:

### CODE REVIEW

File: system/modules/inbox/models/InboxService.php [hunk 1]
```diff
@@ -47,6 +47,7 @@ function addMessage($subject, $message, $user_id = null, $sender_id = null, $par
     }
 
     function sendMail($to, $cc, $bcc, $from, $replyto, $subject, $message) {
+        $this->w->Log->info("Inbox service is asserting retired mail system");
```
Comment (adam-buckley) [inline system/modules/inbox/models/InboxService.php:50]: ```LogService::getInstance($this->w)->```

File: system/modules/task/actions/edit.php [hunk 2]
```diff
@@ -280,7 +280,8 @@ function edit_POST($w)
 
         $messageObject = new Swift_Message("Invite to: " . $task->title);
         $messageObject->setTo([$contact->email]);
-        $messageObject->setFrom($w->Auth->user()->getContact()->email);
+        $messageObject->setReplyTo([$w->Auth->user()->getContact()->email])
```
Comment (adam-buckley) [inline system/modules/task/actions/edit.php:283]: ```AuthService::getInstance($w)->user```

File: system/modules/timelog/models/TimelogService.php [hunk 3]
```diff
@@ -128,7 +128,7 @@ public function getJSTrackingObject() {
 
     public function shouldShowTimer() {
         // Check if tracking object set or existing timelog is running
-        return ($this->w->Auth->user()->hasRole("timelog_user") && ($this->hasTrackingObject() || $this->hasActiveLog()));
+        return ((!empty($this->w->Auth->user())) && $this->w->Auth->user()->hasRole("timelog_user") && ($this->hasTrackingObject() || $this->hasActiveLog()));
```
Comment (adam-buckley) [inline system/modules/timelog/models/TimelogService.php:131]: Also here

**Review by adam-buckley — CHANGES_REQUESTED** [review]
We cannot use the magic __get methods on Web. Once this is fixed I'll approve it.

**Review by strongishllama — APPROVED** [review]

**Pull request merged**